### PR TITLE
Cleanup check primary support

### PR DESCRIPTION
**Pull request by callahad**
See commit shortlog

### CODE REVIEW

File: scripts/check_primary_support [hunk 1]
```diff
@@ -11,11 +11,12 @@ urlp = require('url'),
 util = require('util'),
 fs = require('fs');
 
-primary = require('../lib/primary'),
-logging = require('../lib/logging.js');
-
+const logging = require('../lib/logging.js');
```
Comment (jaredhirsch) [inline scripts/check_primary_support:14]: how about adding a line here to explain why the requires are partitioned? something like

```
// requiring primary sometimes fails, so require & init logging first
```

Comment (callahad) [inline scripts/check_primary_support:14]: I'd rather fix it upstream, but sure. One sec...

**Comment by jaredhirsch** [comment]
I really wish we had unit tests for this, but that's a separate issue.

looks good and works for me:
- eyedeeme: 
  
  ```
  jareds-partybus:callahad-browserid jhirsch$ ./scripts/check_primary_support eyedee.me
  WARNING: Unknown authentication_api.js detected on https://eyedee.me/browserid/sign_in.html
  WARNING: Unknown provisioning_api.js detected on https://eyedee.me/browserid/provision.html
  
  jareds-partybus:mozilla-browserid-testing jhirsch$ ./scripts/check_primary_support eyedee.me
  WARNING: No https://persona.org//authentication_api.js script tag detected
  WARNING: No https://persona.org//provisioning_api.js script tag detected
  ```
- yahoo.login.persona.org:
  
  ```
  jareds-partybus:mozilla-browserid-testing jhirsch$ ./scripts/check_primary_support yahoo.login.persona.org
  https://yahoo.login.persona.org/authentication contains proper script tag
  https://yahoo.login.persona.org/provision contains proper script tag
  
  jareds-partybus:callahad-browserid jhirsch$ ./scripts/check_primary_support yahoo.login.persona.org
  Detected production authentication_api.js on https://yahoo.login.persona.org/authentication
  Detected production provisioning_api.js on https://yahoo.login.persona.org/provision
  ```

**Comment by jaredhirsch** [comment]
well, separate as in: "why don't we have tests for any of these scripts?"

this might be a place to start, but I'm not convinced it's the
highest-value script to test first.

**Pull request merged**